src/definitions/bluebps.json

```diff
@@ -0,0 +1,254 @@
+                        "type": "u16",
+                        "description": "If an individual cell exceeds the configured limit for this duration of time, the power will be locked-out",
+                        "units": "millisecond"
+                    }
+                ]
+            },
+            "current_max": {
+                "id": "9002",
+                "description": "Set the maximum allowed battery current",
+                "payload": [
+                    {
+                        "name": "limit",
+                        "type": "u16",
+                        "description": "The maximum allowed battery current 0~20000 = 0~200A",
+                        "units": "centiamp"
+                    }
+                ]
+            },
+            "current_timeout": {
+                "id": "9003",
+                "description": "Set the over-current timeout",
+                "payload": [
+                    {
+                        "name": "timeout",
+                        "type": "u16",
+                        "description": "If the battery current exceeds the configured limit for this duration of time, the power will be locked-out",
+                        "units": "millisecond"
+                    }
+                ]
+            },
+            "temperature_max": {
+                "id": "9004",
+                "description": "Set the maximum allowed battery temperature",
+                "payload": [
+                    {
+                        "name": "limit",
+                        "type": "u16",
+                        "description": "The minimum voltage allowed for any individual cell. 0~5000: 0~5V",
+                        "units": "centidegrees Centigrade"
+                    }
+                ]
+            },
+            "temperature_timeout": {
+                "id": "9005",
+                "description": "Set the over-temperature timeout",
+                "payload": [
+                    {
+                        "name": "timeout",
+                        "type": "u16",
+                        "description": "If the battery temperature exceeds the configured limit for this duration of time, the power will be locked-out",
+                        "units": "millisecond"
+                    }
+                ]
+            },
+            "stream_rate": {
+                "id": "9006",
+                "description": "Set the frequency to automatically output state messages.",
+                "payload": [
+                    {
+                        "name": "rate",
+                        "type": "u16",
+                        "description": "Rate to stream `state` messages. 0~1000Hz",
+                        "units": "Hz"
+                    }
+                ]
+            },
+            "lpf_setting": {
+                "id": "9007",
+                "description": "Low pass filter setting. 0.0 = no filtering, 0.99 = heavy filtering",
```

Comment:
Can you specify what kind of filter and what this value will change ?
I'm almost sure that you are talking about a 1st order IIR Low-Pass filter, but the documentation need to be explicit about it. 
Also, there is no point to specify the setting in a LP filter if you don't provide the frequency that this filter is running ;)

Reply:
Right now, the low pass filter is like this, I don't know the name for it (maybe alpha-beta)?

```value = value * x + sample * (1-x)```

The lpf setting refers to the value `x` in the filter. I initially called this variable 'lpf alpha', and changed it to this name + description to make it in 'layman's-terms'

The sample frequency is tbd, but we can specify it later. We can fully define these things in the description, I'm just not sure if it's necessary, or if it is, the proper way.

Reply:
Yes, this is a 1st order IIR LP filter, your `x` (also know as α) is a equivalent `RC` factor that is frequency dependent.